README.md

```diff
@@ -1,36 +1,30 @@
-# Sample Twilio-Go App (Twilio Go SDK)
+# Review Rewards Example App (Twilio Go SDK)
 
 An example application made by the Developer Experience - Developer Interfaces team to showcase the usage of the [Twilio Go SDK](https://github.com/twilio/twilio-go), [Programmable Messaging](https://www.twilio.com/docs/sms), and [Programmable Voice](https://www.twilio.com/docs/voice).
 
-Made for the purpose of providing an example application to train individuals on how Golang and the Twilio Go SDK may be used to build a messaging application.
+Made for the purpose of providing an example application to train individuals on how Golang and the Twilio Go SDK may be used to build a web application.
 
 ## Prerequisites
 
 Before installing and running this application, please ensure that you complete the following prerequisites:
 
 - Have a MacOS and Linux environment
-- Install [Docker Desktop](https://www.docker.com/products/docker-desktop/) and [Docker Compose](https://docs.docker.com/compose/install/)
 - Install [Go](https://go.dev/dl/)
 - Install a Code Editor
     - We recommend [Visual Studio Code](https://code.visualstudio.com/) with the [Go Extension](https://marketplace.visualstudio.com/items?itemName=golang.Go) installed
 - Obtain a [Twilio account](https://www.twilio.com/login) and phone number with Messaging and Voice enabled
-- Set the following environment variables using your Twilio account SID, Auth Token, Twilio phone number, desired log level, and localhost tunnel URL (i.e. ngrok tunnel URL):
+- Save your Twilio account SID and Auth Token in the following environment variables:
     - TWILIO_ACCOUNT_SID
     - TWILIO_AUTH_TOKEN
-    - TWILIO_PHONE_NUMBER
-    - LOG_LEVEL
-    - BASE_URL
 - Have [twilio-cli](https://www.twilio.com/docs/twilio-cli/quickstart) installed and logged into your account to easily update your twilio phone number configuration with a single CLI command
 - Obtain access to the [Twilio Console](https://console.twilio.com/)
 - Install/obtain a localhost tunnel to access your localhost server on the internet. We'll use [ngrok](https://ngrok.com/) in our examples.
 
 ## Build the application
 
-To build the application, run `make build-app` to build the application binary in `out/bin/sample-twilio-go`
+To build the application, run `go build`
```

Comment:
Do we need to specify the output binary name here?

Reply:
Changed the build instructions so that we explicitly build a binary named `sample-twilio-go` into a `out/bin` child directory. Instructions to run the application reflect the change as well.